src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/Regex.Tests.Common.cs

```diff
@@ -96,16 +97,13 @@ public static IEnumerable<RegexEngine> AvailableEngines
                         PlatformDetection.IsNotBrowser)
                     {
                         yield return RegexEngine.SourceGenerated;
-
-                        // TODO-NONBACKTRACKING:
-                        // yield return RegexEngine.NonBacktrackingSourceGenerated;
```

Comment:
Do we have an issue tracking sourcegenerated support for nonbacktracking? If we do, do you think it's worth mentioning there that we will want to to add this line back at some point (not commented out of course) or is that just considered obvious?

Reply:
I think it's obvious :-)